newIDE/app/src/fixtures/GDevelopServicesTestData/index.js

```diff
@@ -2783,3 +2786,36 @@ export const fakeAssetPackLicenses = [
     },
   },
 ];
+
+export const fakePromotions: Promotion[] = [
+  {
+    id: '123',
+    imageUrl:
+      'https://resources.gdevelop.io/announcements/GDevelops_Mega_Pack_Updated.png',
+    mobileImageUrl:
+      'https://resources.gdevelop.io/announcements/GDevelops_Mega_Pack_Mobile.jpg',
+    productId: '43994a30-c54b-4f5d-baf5-6e1f99b13824',
+    display: 'all',
+    type: 'asset-pack',
+  },
+  {
+    id: 'premium-featuring-bubble-dogs', // TO REMOVE ON 26/02/24
```

Comment:
Maybe avoid having something in test data that is was not made by GDevelop?

Reply:
yep changing this